src/libstd/dynamic_lib.rs

```diff
@@ -66,21 +66,19 @@ impl DynamicLibrary {
     }
 
     /// Prepends a path to this process's search path for dynamic libraries
-    pub fn prepend_search_path(path: &Path) {
+    pub fn prepend_search_path(path: &path::Path) {
```

Comment:
This type is conventionally imported at the top and just referred to as `Path` instead of `path::Path`